### PR TITLE
[DROOLS-5625] Document unsupported features in Trusty PMML implementation

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca  @jiripetrlik

 see https://issues.redhat.com/browse/DROOLS-5625

This is just a documentation PR.

Please look at "CurrentlyLimitations.md" to see if there is something else (!) missing.

### CODE REVIEW

**Review by danielezonca — APPROVED** [review]

File: kie-pmml-trusty/CurrentLimitations.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Models limitations
+==================
+
+Regression model
+----------------
+1. ModelStats unimplemented
+2. ModelExplanation unimplemented
+3. ModelVerification unimplemented
+
+Tree model
```
Comment (jiripetrlik) [inline kie-pmml-trusty/CurrentLimitations.md:66]: Please provide info also about arrays. Are there any specifics related to it?

File: kie-pmml-trusty/CurrentLimitations.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+6. Surrogate inside CompoundPredicate unimplemented
+7. Only two _XOR_ predicates managed
+8. Only _predictedValue-type_ output field managed
+9. missingValueStrategy partially implemented
+10. missingValuePenalty unimplemented
+11. noTrueChildStrategy partially implemented
+12. splitCharacteristic unimplemented
+13. isScorable unimplemented
+
+Scorecard model
```
Comment (jiripetrlik) [inline kie-pmml-trusty/CurrentLimitations.md:82]: Are there any specifics for reason codes? Is it completely implemented?

**Review by jiripetrlik — COMMENTED** [review]
I have two possible questions. I remember there were some discussions about these topics but I probably forget the conclusion.

File: kie-pmml-trusty/CurrentLimitations.md (hunk 2 above)
Comment (gitgabrio) [reply kie-pmml-trusty/CurrentLimitations.md:82]: @jiripetrlik 
Yes - it should be.

File: kie-pmml-trusty/CurrentLimitations.md (hunk 1 above)
Comment (gitgabrio) [reply kie-pmml-trusty/CurrentLimitations.md:66]: They are implemented: pls look at org.kie.pmml.models.drools.tree.tests.SimpleSetPredicateTreeTest and companion pmml

**Review by jiripetrlik — APPROVED** [review]
@gitgabrio 
Thank you for additional info. LGTM

**Pull request merged**